### ISSUE TITLE
Android 7.1.2 And Lower GSI Support

### ISSUE THREAD

**Issue by Ali382829** [phhusson/treble_experimentations#2666]
Android 7.1.2 And Lower GSI Not Supported I Can't Use Android 7.1.2 GSI
Android 8.1 And Newer GSI Are Supported
Not Possible To Use Android 7.1.2 And Lower GSI
Unfortunately Not Possible To Use Android 7.1.2 And Lower GSI
May Support Android 8.1 And Newer GSI
Not Support Android 7.1.2 And Lower GSI

**Comment by Ali382829** [comment]
Not Support Android 7.1.2 Android Lower GSI

**Comment by NeverSm1le** [comment]
Of course, you need project treble to use GSI and it's only available on android 8

**Comment by Ali382829** [comment]
Only Available On Android 8 GSI

**Comment by ivnvrvnn** [comment]
🙏🙏🙏

**Comment by TechySkills** [comment]
Poetic ahh issue. Blud, listen, There is no project treble below android 8, there is nothing we can do. Either get a developer do hella effort to make a custom rom for your old ahh device, or Just give up, and buy a new phone.

**Comment by Ali382829** [comment]
Require Android 8+ To Use GSI
